rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -174,6 +174,11 @@ namespace rsx
 		{
 			if (offset < 0x100000 /*&& (offset % 0x10) == 0*/)
 			{
+				if (offset != 0x30u)
+				{
+					rsx_log.warning("rsx::get_address(offset=0x%x, location=0x%x, size=0x%x): %s", offset, location, size_to_check, src_loc{line, col, file, func});
```

Comment:
This is silent log spam. If it is unexpected, raise it to error and leave a note about notying developers. That should get us faster feedback if this functionality is misunderstood.

Reply:
Removed.